packages/app/src/app/components/Preview/DevTools/Problems/index.js

```diff
@@ -87,7 +89,10 @@ class Problems extends React.PureComponent<*, State> {
         {root && (
           <div>
             <File>Root</File>
-            {root.map((message, i) => (
+            {/*
+              What's going on here?
+              */
```

Comment:
@CompuIves I couldn't get that component to display, so I wasn't able to integrate it. Under what circumstances is it displayed?

Reply:
Could we get this Root back? This is a reproducible case where it's needed: https://codesandbox.io/s/4061l2nqj0

Reply:
@CompuIves Restored in 03c241d 


![image](https://user-images.githubusercontent.com/13242392/38610781-c7bd938c-3d79-11e8-98be-875ee2f50f9d.png)